rotkehlchen/tests/fixtures/messages.py

```diff
@@ -1,13 +1,43 @@
+from typing import Any, NamedTuple, Optional, Union
 import pytest
+from rotkehlchen.api.websockets.typedefs import WSMessageType
 
 from rotkehlchen.user_messages import MessagesAggregator
 
 
-@pytest.fixture(scope='session')
-def messages_aggregator():
-    return MessagesAggregator()
+class MockedWsMessage(NamedTuple):
+    type: WSMessageType
+    data: Union[dict[str, Any], list[Any]]
+
+
+class MockRotkiNotifier():
+    def __init__(self) -> None:
+        self.messages: list[MockedWsMessage] = []
+
+    def broadcast(  # pylint: disable=unused-argument
+            self,
+            message_type: 'WSMessageType',
+            to_send_data: Union[dict[str, Any], list[Any]],
+            **kwargs: Any,
+    ) -> None:
+        self.messages.append(MockedWsMessage(type=message_type, data=to_send_data))
+
+    def pop_message(self) -> Optional[MockedWsMessage]:
```

Comment:
I would add a function to clear all messages too. Since you added it also as a session scope fixture, it will probably be needed for those tests that use it in a session scope. Otherwise it will have leftover messages from other tests whenever it's used in a test